### DISCUSSION TITLE
Bootloop with USE_SHUTTER_ESP32 and more than 4 shutters

### DISCUSSION THREAD

**Discussion by roschuba** [arendst/Tasmota#18651] [Q&A]
Hi @stefanbode,

I'm not new to Tasmota-Shutters, I have many shutters controlled by Shelly 2.5 already running over a year.

Now I built a controller for 8 shutters to replace my over 20 years old original Velux IR Controllers.

I'm using an ESP32-DevKit to operate the relays and I compiled tasmota 12.5.0 with option USE_SHUTTER_ESP32.

Everything works fine up to 4 shutters defined.

When I send the command 'ShutterRelay5 9' to enable the 5th shutter using relays 9 and 10 and reboot tasmota then tasmota goes in a boot loop showing the following on the console:

`Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:184
load:0x40078000,len:12680
load:0x40080400,len:2908
entry 0x400805c4

00:00:00.002 HDW: ESP32-D0WDQ6
00:00:00.047 UFS: FlashFS mounted with 300 kB free
00:00:00.069 CFG: Loaded from File, Anzahl 152
00:00:00.078 QPC: Count 1
00:00:00.206 BRY: Berry initialized, RAM used=4041 bytes
00:00:00.222 TFS: File '.drvset003' not found
00:00:00.223 CFG: Energy use defaults as file system not ready or file not found
00:00:00.224 Shutter: Standard verwenden
00:00:00.234 SHUTTER: About to load settings from file /.drvset027
00:00:00.268 SHT: ShutterMode: 2
00:00:00.269 SHT: ShutterMode: 2
00:00:00.271 SHT: ShutterMode: 2
00:00:00.273 SHT: ShutterMode: 2
00:00:00.275 SHT: ShutterMode: 2
00:00:00.287 Projekt tasmota - WG_VeluxShutter1 Version 12.5.0(shutter)-2_0_7(2023.05.10 20:50:09)
00:00:00.940 WIF: verbinden mit AP1 xxxxxxxx Channel 11 BSSId 02:EC:DA:FD:7F:C6 in Modus 11n als WG-VeluxShutter1-1028...
00:00:02.695 WIF: verbunden
00:00:02.953 HTP: Web-Server aktiv bei WG-VeluxShutter1-1028 mit IP-Adresse 192.168.60.232
11:07:52.945 MQT: Verbindungsversuch...
11:07:53.060 MQT: verbunden
11:07:53.064 MQT: tele/WG_VeluxShutter1/LWT = Online (beibehalten)
11:07:53.067 MQT: cmnd/WG_VeluxShutter1/POWER =
11:07:53.072 MQT: tele/WG_VeluxShutter1/INFO1 = {"Info1":{"Module":"ESP32-DevKit","Version":"12.5.0(shutter)","FallbackTopic":"cmnd/DVES_F36404_fb/","GroupTopic":"cmnd/tasmotas/"}}
11:07:53.089 MQT: tele/WG_VeluxShutter1/INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"WG-VeluxShutter1-1028","IPAddress":"192.168.60.232","IP6Global":"","IP6Local":"fe80::2662:abff:fef3:6404"}}
11:07:53.104 MQT: tele/WG_VeluxShutter1/INFO3 = {"Info3":{"RestartReason":{"Exception":3,"Reason":"LoadStoreError","EPC":"401189d9","EXCVADDR":"40000000","CallChain":["401189d6","40100505","4012bf3e","4013c69e","400e51f9"]},"BootCount":138}}
11:07:53.130 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER1":"OFF"}
11:07:53.136 MQT: stat/WG_VeluxShutter1/POWER1 = OFF
11:07:53.138 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER2":"OFF"}
11:07:53.140 MQT: stat/WG_VeluxShutter1/POWER2 = OFF
11:07:53.152 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER3":"OFF"}
11:07:53.155 MQT: stat/WG_VeluxShutter1/POWER3 = OFF
11:07:53.157 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER4":"OFF"}
11:07:53.169 MQT: stat/WG_VeluxShutter1/POWER4 = OFF
11:07:53.171 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER5":"OFF"}
11:07:53.173 MQT: stat/WG_VeluxShutter1/POWER5 = OFF
11:07:53.179 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER6":"OFF"}
11:07:53.181 MQT: stat/WG_VeluxShutter1/POWER6 = OFF
11:07:53.197 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER7":"OFF"}
11:07:53.199 MQT: stat/WG_VeluxShutter1/POWER7 = OFF
11:07:53.201 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER8":"OFF"}
11:07:53.216 MQT: stat/WG_VeluxShutter1/POWER8 = OFF
11:07:53.224 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER9":"OFF"}
11:07:53.226 MQT: stat/WG_VeluxShutter1/POWER9 = OFF
11:07:53.231 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER10":"OFF"}
11:07:53.233 MQT: stat/WG_VeluxShutter1/POWER10 = OFF
11:07:53.250 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER11":"OFF"}
11:07:53.253 MQT: stat/WG_VeluxShutter1/POWER11 = OFF
11:07:53.256 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER12":"OFF"}
11:07:53.272 MQT: stat/WG_VeluxShutter1/POWER12 = OFF
11:07:53.279 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER13":"OFF"}
11:07:53.282 MQT: stat/WG_VeluxShutter1/POWER13 = OFF
11:07:53.284 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER14":"OFF"}
11:07:53.298 MQT: stat/WG_VeluxShutter1/POWER14 = OFF
11:07:53.300 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER15":"OFF"}
11:07:53.303 MQT: stat/WG_VeluxShutter1/POWER15 = OFF
11:07:53.305 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER16":"OFF"}
11:07:53.321 MQT: stat/WG_VeluxShutter1/POWER16 = OFF
11:07:55.925 QPC: Reset
11:07:56.931 MQT: tele/WG_VeluxShutter1/STATE = {"Time":"2023-05-12T11:07:56","Uptime":"0T00:00:08","UptimeSec":8,"Heap":166,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Berry":{"HeapUsed":3,"Objects":44},"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"OFF","POWER7":"OFF","POWER8":"OFF","POWER9":"OFF","POWER10":"OFF","POWER11":"OFF","POWER12":"OFF","POWER13":"OFF","POWER14":"OFF","POWER15":"OFF","POWER16":"OFF","Wifi":{"AP":1,"SSId":"xxxxxx","BSSId":"02:EC:DA:FD:7F:C6","Channel":11,"Mode":"11n","RSSI":100,"Signal":-43,"LinkCount":1,"Downtime":"0T00:00:03"}}
11:07:56.999 MQT: tele/WG_VeluxShutter1/SENSOR = {"Time":"2023-05-12T11:07:56","Shutter1":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter2":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter3":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter4":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter5":{"Position":0,"Direction":0,"Target":0,"Tilt":0}}
Guru Meditation Error: Core  1 panic'ed (LoadStoreError). Exception was unhandled.

Core  1 register dump:
PC      : 0x401189d9  PS      : 0x00060f30  A0      : 0x80100508  A1      : 0x3ffb2040  
A2      : 0x00000000  A3      : 0x3ffc4a68  A4      : 0x3f402ad7  A5      : 0x00000000
A6      : 0x3f424408  A7      : 0x3ffc2f34  A8      : 0x40000000  A9      : 0x00046e98  
A10     : 0x0001b92f  A11     : 0x3f424408  A12     : 0x00000000  A13     : 0x00000000
A14     : 0x00000e78  A15     : 0x00000000  SAR     : 0x0000001c  EXCCAUSE: 0x00000003  
EXCVADDR: 0x40000000  LBEG    : 0x4008d859  LEND    : 0x4008d869  LCOUNT  : 0xffffffff

Backtrace: 0x401189d6:0x3ffb2040 0x40100505:0x3ffb21e0 0x4012bf3e:0x3ffb2200 0x4013c69e:0x3ffb2220 0x400e51f9:0x3ffb2290

ELF file SHA256: 0000000000000000

Rebooting...`

I already tried different ESP32-Devkits to make sure it's not a hardware problem. 
The problem persists even if nothing is connected to the ESP32-Devkit exept the USB-cable.

**Comment by Jason2866** [comment]
Can you try latest development? There are a few changes in shutter since 12.5.0

**Reply by roschuba** [reply]
Thank you @Jason2866 for your fast response.

I downloaded the current development software and compiled my binary.

But the problem persists:

```
22:25:50.174 APP: starte neu
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:184
load:0x40078000,len:12680
load:0x40080400,len:2908
entry 0x400805c4

00:00:00.003 HDW: ESP32-D0WDQ6
00:00:00.041 UFS: FlashFS mounted with 300 kB free
00:00:00.072 CFG: Loaded from File, Anzahl 179
00:00:00.082 QPC: Count 1
00:00:00.204 BRY: Berry initialized, RAM used=3922 bytes
00:00:00.220 TFS: File '.drvset003' not found
00:00:00.221 CFG: Energy use defaults as file system not ready or file not found
00:00:00.222 Shutter: Standard verwenden
00:00:00.233 SHT: About to load settings from file /.drvset027
00:00:00.258 SHT: ShutterMode: 2
00:00:00.259 SHT: ShutterMode: 2
00:00:00.261 SHT: ShutterMode: 2
00:00:00.263 SHT: ShutterMode: 2
00:00:00.264 SHT: ShutterMode: 2
00:00:00.275 Projekt tasmota - WG_VeluxShutter1 Version 12.5.0.2(shutter)-2_0_9(2023.05.13 22:13:03)
00:00:00.972 WIF: verbinden mit AP1 xxxxxxxx Channel 11 BSSId 02:EC:DA:FD:7F:C6 in Modus 11n als WG-VeluxShutter1-1028...
00:00:02.674 WIF: verbunden
00:00:02.933 HTP: Web-Server aktiv bei WG-VeluxShutter1-1028 mit IP-Adresse 192.168.60.232
22:25:54.906 MQT: Verbindungsversuch...
22:25:54.985 MQT: verbunden
22:25:55.000 MQT: tele/WG_VeluxShutter1/LWT = Online (beibehalten)
22:25:55.003 MQT: cmnd/WG_VeluxShutter1/POWER =
22:25:55.009 MQT: tele/WG_VeluxShutter1/INFO1 = {"Info1":{"Module":"ESP32-DevKit","Version":"12.5.0.2(shutter)","FallbackTopic":"cmnd/DVES_F36404_fb/","GroupTopic":"cmnd/tasmotas/"}}
22:25:55.025 MQT: tele/WG_VeluxShutter1/INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"WG-VeluxShutter1-1028","IPAddress":"192.168.60.232","IP6Global":"","IP6Local":"fe80::2662:abff:fef3:6404"}}
22:25:55.039 MQT: tele/WG_VeluxShutter1/INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":153}}
22:25:55.053 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER1":"OFF"}
22:25:55.056 MQT: stat/WG_VeluxShutter1/POWER1 = OFF
22:25:55.058 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER2":"OFF"}
22:25:55.071 MQT: stat/WG_VeluxShutter1/POWER2 = OFF
22:25:55.073 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER3":"OFF"}
22:25:55.075 MQT: stat/WG_VeluxShutter1/POWER3 = OFF
22:25:55.077 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER4":"OFF"}
22:25:55.090 MQT: stat/WG_VeluxShutter1/POWER4 = OFF
22:25:55.096 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER5":"OFF"}
22:25:55.099 MQT: stat/WG_VeluxShutter1/POWER5 = OFF
22:25:55.101 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER6":"OFF"}
22:25:55.114 MQT: stat/WG_VeluxShutter1/POWER6 = OFF
22:25:55.117 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER7":"OFF"}
22:25:55.119 MQT: stat/WG_VeluxShutter1/POWER7 = OFF
22:25:55.121 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER8":"OFF"}
22:25:55.134 MQT: stat/WG_VeluxShutter1/POWER8 = OFF
22:25:55.136 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER9":"OFF"}
22:25:55.138 MQT: stat/WG_VeluxShutter1/POWER9 = OFF
22:25:55.151 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER10":"OFF"}
22:25:55.158 MQT: stat/WG_VeluxShutter1/POWER10 = OFF
22:25:55.160 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER11":"OFF"}
22:25:55.163 MQT: stat/WG_VeluxShutter1/POWER11 = OFF
22:25:55.165 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER12":"OFF"}
22:25:55.178 MQT: stat/WG_VeluxShutter1/POWER12 = OFF
22:25:55.180 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER13":"OFF"}
22:25:55.182 MQT: stat/WG_VeluxShutter1/POWER13 = OFF
22:25:55.195 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER14":"OFF"}
22:25:55.199 MQT: stat/WG_VeluxShutter1/POWER14 = OFF
22:25:55.202 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER15":"OFF"}
22:25:55.208 MQT: stat/WG_VeluxShutter1/POWER15 = OFF
22:25:55.211 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER16":"OFF"}
22:25:55.214 MQT: stat/WG_VeluxShutter1/POWER16 = OFF
22:25:57.913 QPC: Reset
22:25:58.916 MQT: tele/WG_VeluxShutter1/STATE = {"Time":"2023-05-13T22:25:58","Uptime":"0T00:00:08","UptimeSec":8,"Heap":166,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Berry":{"HeapUsed":3,"Objects":40},"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"OFF","POWER7":"OFF","POWER8":"OFF","POWER9":"OFF","POWER10":"OFF","POWER11":"OFF","POWER12":"OFF","POWER13":"OFF","POWER14":"OFF","POWER15":"OFF","POWER16":"OFF","Wifi":{"AP":1,"SSId":"xxxxxxxx","BSSId":"02:EC:DA:FD:7F:C6","Channel":11,"Mode":"11n","RSSI":100,"Signal":-44,"LinkCount":1,"Downtime":"0T00:00:03"}}
22:25:58.983 MQT: tele/WG_VeluxShutter1/SENSOR = {"Time":"2023-05-13T22:25:58","Shutter1":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter2":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter3":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter4":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter5":{"Position":100,"Direction":0,"Target":100,"Tilt":0}}
Guru Meditation Error: Core  1 panic'ed (LoadStoreError). Exception was unhandled.

Core  1 register dump:
PC      : 0x40119281  PS      : 0x00060f30  A0      : 0x8010070c  A1      : 0x3ffb2040  
A2      : 0x00000000  A3      : 0x3ffc4a88  A4      : 0x3f402b8b  A5      : 0x00000000
A6      : 0x3f4247db  A7      : 0x3ffc2e44  A8      : 0x40000000  A9      : 0x00046e98  
A10     : 0x0001b92f  A11     : 0x3f4247db  A12     : 0x00000000  A13     : 0x00000000
A14     : 0x00000e78  A15     : 0x00000000  SAR     : 0x00000019  EXCCAUSE: 0x00000003  
EXCVADDR: 0x40000000  LBEG    : 0x4008d9c5  LEND    : 0x4008d9d5  LCOUNT  : 0xffffffff


Backtrace: 0x4011927e:0x3ffb2040 0x40100709:0x3ffb21e0 0x4012c322:0x3ffb2200 0x4013d746:0x3ffb2220 0x400e5389:0x3ffb2290




ELF file SHA256: 0000000000000000

Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:184
load:0x40078000,len:12680
load:0x40080400,len:2908
entry 0x400805c4

00:00:00.003 HDW: ESP32-D0WDQ6
00:00:00.045 UFS: FlashFS mounted with 300 kB free
00:00:00.066 CFG: Loaded from File, Anzahl 180
00:00:00.076 QPC: Count 1
00:00:00.198 BRY: Berry initialized, RAM used=3922 bytes
00:00:00.215 TFS: File '.drvset003' not found
00:00:00.215 CFG: Energy use defaults as file system not ready or file not found
00:00:00.216 Shutter: Standard verwenden
00:00:00.227 SHT: About to load settings from file /.drvset027
00:00:00.254 SHT: ShutterMode: 2
00:00:00.255 SHT: ShutterMode: 2
00:00:00.257 SHT: ShutterMode: 2
00:00:00.259 SHT: ShutterMode: 2
00:00:00.260 SHT: ShutterMode: 2
00:00:00.271 Projekt tasmota - WG_VeluxShutter1 Version 12.5.0.2(shutter)-2_0_9(2023.05.13 22:13:03)
00:00:00.964 WIF: verbinden mit AP1 xxxxxxxx Channel 11 BSSId 02:EC:DA:FD:7F:C6 in Modus 11n als WG-VeluxShutter1-1028...
00:00:02.670 WIF: verbunden
00:00:02.929 HTP: Web-Server aktiv bei WG-VeluxShutter1-1028 mit IP-Adresse 192.168.60.232
22:26:38.940 MQT: Verbindungsversuch...
22:26:39.081 MQT: verbunden
22:26:39.085 MQT: tele/WG_VeluxShutter1/LWT = Online (beibehalten)
22:26:39.087 MQT: cmnd/WG_VeluxShutter1/POWER =
22:26:39.094 MQT: tele/WG_VeluxShutter1/INFO1 = {"Info1":{"Module":"ESP32-DevKit","Version":"12.5.0.2(shutter)","FallbackTopic":"cmnd/DVES_F36404_fb/","GroupTopic":"cmnd/tasmotas/"}}
22:26:39.110 MQT: tele/WG_VeluxShutter1/INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"WG-VeluxShutter1-1028","IPAddress":"192.168.60.232","IP6Global":"","IP6Local":"fe80::2662:abff:fef3:6404"}}
22:26:39.126 MQT: tele/WG_VeluxShutter1/INFO3 = {"Info3":{"RestartReason":{"Exception":3,"Reason":"LoadStoreError","EPC":"40119281","EXCVADDR":"40000000","CallChain":["4011927e","40100709","4012c322","4013d746","400e5389"]},"BootCount":154}}
22:26:39.150 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER1":"OFF"}
22:26:39.152 MQT: stat/WG_VeluxShutter1/POWER1 = OFF
22:26:39.154 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER2":"OFF"}
22:26:39.167 MQT: stat/WG_VeluxShutter1/POWER2 = OFF
22:26:39.169 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER3":"OFF"}
22:26:39.172 MQT: stat/WG_VeluxShutter1/POWER3 = OFF
22:26:39.174 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER4":"OFF"}
22:26:39.186 MQT: stat/WG_VeluxShutter1/POWER4 = OFF
22:26:39.189 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER5":"OFF"}
22:26:39.191 MQT: stat/WG_VeluxShutter1/POWER5 = OFF
22:26:39.203 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER6":"OFF"}
22:26:39.206 MQT: stat/WG_VeluxShutter1/POWER6 = OFF
22:26:39.208 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER7":"OFF"}
22:26:39.221 MQT: stat/WG_VeluxShutter1/POWER7 = OFF
22:26:39.223 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER8":"OFF"}
22:26:39.226 MQT: stat/WG_VeluxShutter1/POWER8 = OFF
22:26:39.228 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER9":"OFF"}
22:26:39.242 MQT: stat/WG_VeluxShutter1/POWER9 = OFF
22:26:39.245 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER10":"OFF"}
22:26:39.247 MQT: stat/WG_VeluxShutter1/POWER10 = OFF
22:26:39.250 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER11":"OFF"}
22:26:39.262 MQT: stat/WG_VeluxShutter1/POWER11 = OFF
22:26:39.265 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER12":"OFF"}
22:26:39.267 MQT: stat/WG_VeluxShutter1/POWER12 = OFF
22:26:39.280 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER13":"OFF"}
22:26:39.282 MQT: stat/WG_VeluxShutter1/POWER13 = OFF
22:26:39.285 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER14":"OFF"}
22:26:39.298 MQT: stat/WG_VeluxShutter1/POWER14 = OFF
22:26:39.300 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER15":"OFF"}
22:26:39.303 MQT: stat/WG_VeluxShutter1/POWER15 = OFF
22:26:39.305 MQT: stat/WG_VeluxShutter1/RESULT = {"POWER16":"OFF"}
22:26:39.318 MQT: stat/WG_VeluxShutter1/POWER16 = OFF
22:26:41.938 QPC: Reset
22:26:42.942 MQT: tele/WG_VeluxShutter1/STATE = {"Time":"2023-05-13T22:26:42","Uptime":"0T00:00:08","UptimeSec":8,"Heap":165,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Berry":{"HeapUsed":3,"Objects":40},"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"OFF","POWER7":"OFF","POWER8":"OFF","POWER9":"OFF","POWER10":"OFF","POWER11":"OFF","POWER12":"OFF","POWER13":"OFF","POWER14":"OFF","POWER15":"OFF","POWER16":"OFF","Wifi":{"AP":1,"SSId":"xxxxxxxx","BSSId":"02:EC:DA:FD:7F:C6","Channel":11,"Mode":"11n","RSSI":100,"Signal":-49,"LinkCount":1,"Downtime":"0T00:00:03"}}
22:26:43.004 MQT: tele/WG_VeluxShutter1/SENSOR = {"Time":"2023-05-13T22:26:42","Shutter1":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter2":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter3":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter4":{"Position":0,"Direction":0,"Target":0,"Tilt":0},"Shutter5":{"Position":100,"Direction":0,"Target":100,"Tilt":0}}
Guru Meditation Error: Core  1 panic'ed (LoadStoreError). Exception was unhandled.

Core  1 register dump:
PC      : 0x40119281  PS      : 0x00060f30  A0      : 0x8010070c  A1      : 0x3ffb2040  
A2      : 0x00000000  A3      : 0x3ffc4a88  A4      : 0x3f402b8b  A5      : 0x00000000  
A6      : 0x3f4247db  A7      : 0x3ffc2e44  A8      : 0x40000000  A9      : 0x00046e98  
A10     : 0x0001b92f  A11     : 0x3f4247db  A12     : 0x00000000  A13     : 0x00000000  
A14     : 0x00000e78  A15     : 0x00000000  SAR     : 0x0000001c  EXCCAUSE: 0x00000003  
EXCVADDR: 0x40000000  LBEG    : 0x4008d9c5  LEND    : 0x4008d9d5  LCOUNT  : 0xffffffff  


Backtrace: 0x4011927e:0x3ffb2040 0x40100709:0x3ffb21e0 0x4012c322:0x3ffb2200 0x4013d746:0x3ffb2220 0x400e5389:0x3ffb2290




ELF file SHA256: 0000000000000000

Rebooting...


```

**Comment by Jason2866** [comment]
@stefanbode Can you have a look?

**Reply by stefanbode** [reply]
Yes for sure I will do. Please be patient. Can you check if you have in the filesytem under the system files a .drv0027 or similar. You can only use 5+ shutter if there is a filesystem available. Anyhow it should not crash

**Reply by roschuba** [reply]
Yes, the file is visible in the filesystem

![grafik](https://github.com/arendst/Tasmota/assets/67481611/bc0e3ada-6ac4-4619-a104-af0a40d4f821)

**Comment by stefanbode** [comment]
I can reproduce the error in the autoconfig announcement. This will stay on 4 shutters currently not to increase the message. but it seems to out of bound. a bit strange.

**Reply by stefanbode** [reply]
Submitted a fix https://github.com/arendst/Tasmota/pull/18673
